templates/.github/workflows/go-check.yml

```diff
@@ -48,4 +56,19 @@ jobs:
           run: |
             set -o pipefail
             staticcheck ./... | sed -e 's@\(.*\)\.go@./\1.go@g'
+      - name: go generate
+        uses: protocol/multiple-go-modules@v1.2
+        if: ${{ env.RUNGOGENERATE == 'true' }}
```

Comment:
If we only set this variable when true, I _think_ we could check `${{ env.RUNGOGENERATE }}` and drop the `== true`.

Reply:
Unfortunately that doesn't work. We can omit the `${{` though.

Reply:
Oh well.